core/new-gui/src/app/dashboard/user/component/user-project/user-project.component.ts

```diff
@@ -94,6 +94,7 @@ export class UserProjectComponent implements OnInit {
   }
 
   private getUserProjectArray() {
+    this.userProjectService.refreshProjectList();
```

Comment:
`refreshProjectList` uses `this.retrieveProjectList()` internally, and line 99 calls `retrieveProjectList()` explicitly. Isn't this redundant? also the result of `refreshProjectList` is an observable which is not being subscribed, is that intentional?

Reply:
I also feel this is unnecessarily complicated. @fjch1997 can you merge the two functions into one?